processor/servicegraphprocessor/processor.go

```diff
@@ -204,7 +204,7 @@ func (p *processor) aggregateMetrics(ctx context.Context, td ptrace.Traces) (err
 						e.ClientService = serviceName
 						e.ClientLatencySec = float64(span.EndTimestamp()-span.StartTimestamp()) / float64(time.Millisecond.Nanoseconds())
 						e.Failed = e.Failed || span.Status().Code() == ptrace.StatusCodeError
-						p.upsertDimensions(e.Dimensions, rAttributes, span.Attributes())
+						p.upsertDimensions(strings.ToLower(ptrace.SpanKindClient.String()), e.Dimensions, rAttributes, span.Attributes())
```

Comment:
Can we use `client` and `server` instead of this? `SPAN_KIND_CLIENT` is extremely verbose in my opinion.

Reply:
Failed to mention that it should properly tag consumers, producers and dbs as well.

Reply:
consumers, producers and dbs's tag are already well handled by `fallthrough`

Reply:
Yes, but they should be tagged with their correct value if possible, not `client` and `server`, I think.

Reply:
There is a  `connection_type ` label to show whitch type of conn, I don't think we need specific it in attribute label. The prefix works for distinguishing the attributes from which side.
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/3b838e92cb6a167eaf7d4bbb2b09793e4e1b93c0/processor/servicegraphprocessor/processor.go#L334-L344

Reply:
I prefer that we use the most precise prefix, but don't want to block the PR just for that. I'm fine with `client` and `server`.